components/agents/org.wso2.carbon.identity.sso.agent/src/main/java/org/wso2/carbon/identity/sso/agent/oauth2/SAML2GrantAccessTokenRequestor.java

```diff
@@ -103,6 +118,11 @@ public static String executePost(String targetURL, String urlParameters, String
             rd.close();
             return response.toString();
 
+        } catch (Exception e) {
+            if (log.isDebugEnabled()) {
+                log.debug("Exception occurred : ", e);
+            }
+            throw new SSOAgentException(e);
```

Comment:
no need to put debug log when throwing exception. Put a meaningful message when throwing exception 

ex. throw new Exception("msg",e)